frontend/src/components/details/DetailsComponent.js

```diff
@@ -38,21 +39,82 @@ class DetailsComponent extends Component {
+                        loadComparePage={this.state.compareCanvas}
+                        onCompareClick={handleCompareButton}
+                        scenarioName={scenarioNames.modified}
+                    />
+                    <h2 style={styleSubtitle}>The chosen Model is {selectedProduct.modelName}</h2>
+                    <div style={{ marginLeft: 15 }}>
+                        <SelectVariable loadComparePage={this.state.compareCanvas} />
+                    </div>
+
+                    <Canvas loadComparePage={this.state.compareCanvas} />
+                </React.Fragment>
+            );
+        } else {
+            return (
+                <Container fluid={true}>
+                    <Row>
+                        <div
+                            style={{
+                                width: 15,
+                            }}>
+                        </div>
+                        <Col
+                            style={{ backgroundColor: 'white' }}
+                        >
+                            <DividerPanel
+                                loadComparePage={this.state.compareCanvas}
+                                onCompareClick={handleCompareButton}
+                                scenarioName={scenarioNames.baseline}
+                            />
+                            <h2 style={styleSubtitle}>The chosen Model is {selectedProduct.modelName}</h2>
+                            <div style={{ marginLeft: 15 }}>
+                                <SelectVariable loadComparePage={this.state.compareCanvas} />
+                            </div>
+
+                            <Canvas loadComparePage={this.state.compareCanvas} />
+                        </Col>
+                        <div
+                            style={{
+                                width: 15,
+                            }}>
+                        </div>
+                        <Col
+                            style={{ backgroundColor: 'white' }}
+                        >
+                            <DividerPanel
+                                loadComparePage={this.state.compareCanvas}
+                                onCompareClick={handleCompareButton}
+                                scenarioName={scenarioNames.modified}
+                            />
+                            <h2 style={styleSubtitle}>The chosen Model is {selectedProduct.modelName}</h2>
+                            <div style={{ marginLeft: 15 }}>
+                                <SelectVariable loadComparePage={this.state.compareCanvas} />
+                            </div>
+
+                            <Canvas loadComparePage={this.state.compareCanvas} />
+                        </Col>
+                        <div
+                            style={{
+                                width: 15,
+                            }}>
+                        </div>
+                    </Row>
+                </Container>
+            );
```

Comment:
This doesn't quite seem to work for my Desktop screen when on 100% Zoom, maybe you would want to tweak it a little bit more.

100% Zoom:
![image](https://user-images.githubusercontent.com/22003632/120181354-38e7a280-c20d-11eb-9961-25ccc332fca0.png)


90% Zoom:
![image](https://user-images.githubusercontent.com/22003632/120181384-469d2800-c20d-11eb-8fd8-26ae1e7f864b.png)

Reply:
Also, the Labels from the legend of the pie chart component are a little bit out of bounds, and the labels of the bar chart are a little bit cut. @gandompm

Reply:
Favorites and add button looks good :) Labels of the pie chart are also showing outside of the frame for me

Reply:
> Also, the Labels from the legend of the pie chart component are a little bit out of bounds, and the labels of the bar chart are a little bit cut. @gandompm

ok then let me see how the pie chart Issue can be resolved

Reply:
> This doesn't quite seem to work for my Desktop screen when on 100% Zoom, maybe you would want to tweak it a little bit more.
> 
> 100% Zoom:
> ![image](https://user-images.githubusercontent.com/22003632/120181354-38e7a280-c20d-11eb-9961-25ccc332fca0.png)
> 
> 90% Zoom:
> ![image](https://user-images.githubusercontent.com/22003632/120181384-469d2800-c20d-11eb-8fd8-26ae1e7f864b.png)

for me is the same problem. also in 90% of zoom scenarios are not side by side.